### DISCUSSION TITLE
将 CSS 变量写在每一个组件的 style 里实在太不优雅了

### DISCUSSION THREAD

**Discussion by doooooit** [tusen-ai/naive-ui#3375] [General]
![image](https://user-images.githubusercontent.com/11793795/181468069-28583d59-0ff2-4def-8e5a-9c2771e3bed0.png)

如图，一眼望过去全是 CSS 变量，丑陋，而且给调试也增加了虽然微小但确实存在的不便。
而且每一个组件都写一遍，重复的内容难道不会徒增资源占用吗

**Comment by 07akioni** [comment]
这个可以关掉，看 config provider

<img width="958" alt="image" src="https://user-images.githubusercontent.com/18677354/181474926-1604b7b0-b0bc-47b7-a230-3a813cfb958c.png">

其次对于性能的推测不一定是可靠的，真正可靠的是实际测量的表现，目前没有观测到对性能有什么负面影响

**Reply by doooooit** [reply]
感谢解答
